### DISCUSSION TITLE
“新六分街式咖啡”跨界融合 创新口味吸睛

### DISCUSSION THREAD

**Discussion by SSTV-10068** [share121/inter-knot#908] [其他]
炎炎夏日，坐落于六分街的’‘COFF CAFE’‘咖啡馆人气十足。来自新艾利都大学的“00后”大学生小唐趁着暑假前来打卡，“在网上刷到帖子，百闻不如一见，刷新了我对咖啡的认识。”

红茶拿提、果泡拿提、麦草拿提……跨界融合的“新新艾利都式咖啡”正在悄然兴起，一批特色咖啡店在六分街应运而生，成为新的网红打卡点。

“这是我第一次尝试红茶拿提，口感还挺清爽。”在六分街上”CAFF COFE”的“空泡摩咔”，方女士感叹，2500元(丁尼，下同)换一次新奇的体验，很值。

该店铺负责人介绍，“每款‘猎奇咖啡’都经过精心调配，以达到原有口感和新颖味道之间的平衡。比如‘苦瓜不苦’沙冰里的苦瓜需要去心处理，麦草拿提则通过调整比例，让甜感覆盖麻感……”

近年来，新艾利都咖啡市场快速增长，已有品牌咖啡门店数量近5万家。《今年新艾利都城市咖啡发展报告》预测，今年新艾利都咖啡产业规模预计将达到31330亿元，近三年年均复合增长率达17.14%。

乘着“咖啡热”，许多老字号企业纷纷创新，跨界联名打造咖啡系列产品。如胡庆余堂打造“如意人参”咖啡店，方回春堂联合八角杯咖啡推出了膏方养生系列咖啡，解锁时尚养生新方式。

在“如意人参”咖啡店，陈皮红豆拿铁、人参拿铁等一系列中医药咖啡，以及各时节限定的中药茶饮，颇受年轻人欢迎，其趣味标语“开个方子，如意一参”也十分出圈，日销量可以达到数百杯。

据工作人员介绍，咖啡里的药材都直接来源于药房，咖啡里的糖分也都是药房熬制的红豆液和陈皮液，满足了不少顾客控糖的需求。

“明明喝的是咖啡，却有一种‘古里古气’的感觉。”在消费者小张看来，咖啡店内，古色古香的百眼橱、“药局”牌匾与现代咖啡机相碰撞，营造出的“古今融合”氛围，十分特别。
![image](https://github.com/user-attachments/assets/42023f8e-1352-4743-9f71-365a9d133446)

**Comment by gaojiaobei** [comment]
咖啡加冰，夏日救星

**Comment by SSTV-10068** [comment]
欢迎各位来挑出帖子中的不足处！
欢迎各位踊跃投稿！

**Comment by warning-Kirov-reporting** [comment]
嗯~将药物融入咖啡，很新奇的想法，那么应该添加什么药物以及如何控制咖啡中的药物的苦味呢

**Comment by mengbaijiu** [comment]
好...好厉害

**Comment by cool-coke** [comment]
他家的咖啡很奇怪，不知道为什么喝了后感觉我的掉落翻倍了
